### ISSUE TITLE
Add events metrics on soak reports

### ISSUE THREAD

**Issue by fuchsnj** [vectordotdev/vector#10001]
Right now the soak report includes metrics in bytes/sec. It would also be useful to have events/sec.

**Comment by jszwedko** [comment]
Agreed, just noting that it'd be nice to have both measures rather than replacing bytes with events.

**Comment by blt** [comment]
> Agreed, just noting that it'd be nice to have both measures rather than replacing bytes with events.

Good call out, and agreed. Unless there's some extraordinary circumstance the metrics we capture from the soaks are append-only.
